company/careers.md

```diff
@@ -45,6 +45,12 @@ We're hiring! Check out our open positions:
 - [Finance & Operations Analyst](https://jobs.lever.co/sourcegraph/35d6628e-9e9a-4385-a4ea-67e411732cb4)
 - [VP Talent](../handbook/talent/vp_talent.md)
```

Comment:
```suggestion
```

Reply:
Removing VP Talent from the Operations section if we have a separate People Ops header